www-apps/radarr/files/radarr.init

```diff
@@ -0,0 +1,48 @@
+#!/sbin/openrc-run
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+depend() {
+        need net
+}
+
+run_dir="/var/run/radarr"
+pidfile="$run_dir/radarr.pid"
+etc_dir="/etc/radarr"
+var_dir="/var/lib/radarr"
+log_dir="/var/log/radarr"
+log_file="radarr.log"
+program="$APP_DIR/Radarr"
+
+check_config() {
+    
+    if [ ! -d "${run_dir}" ]; then
+            mkdir "${run_dir}"
+    fi
+
+    # Permission stuff. Should ensure that the daemon user always have write permissions.
+    # Quick and ugly but should get the job done.
+
+    chown -R ${USER}:${GROUP} "${run_dir}"
```

Comment:
I'd avoid doing this. Use checkpath and tmpfiles instead (tmpfiles in the ebuild). We've had a lot of sec bugs from stuff like this

Reply:
I'm not an openrc user (I used systemd), so this openrc stuff is pretty rusty for me, and I can't test it.

I used `checkpath`; do we need to use `tmpfiles`? If so, how exactly should that be done? PR's greatly appreciated :)

Reply:
I greatly simplified the openrc scripts which I believe eliminate this concern.